.github/workflows/main.yml

```diff
@@ -158,9 +145,9 @@ jobs:
       - name: Maven Install
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl '${{ matrix.modules }}'
+          ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl $(echo '${{ matrix.modules }}' | cut -d' ' -f1)
       - name: Maven Tests
-        run: ./mvnw test -B -e -Dair.check.skip-all -pl '${{ matrix.modules }}'
+        run: ./mvnw test -B -e -Dair.check.skip-all -pl ${{ matrix.modules }}
```

Comment:
that's a hack... at least rename modules to `modules-with-configuration` 

can `modules` be a JSON?

Reply:
I'd prefer not to use a long name since it makes the lines longer and harder to read. The code is short so it's easy to see what's happening.

It's possible that it can be an object, but I couldn't find any examples. I'll merge this and see if we improve it.